getting_started/install_openshift.adoc

```diff
@@ -0,0 +1,229 @@
+[[getting-started-install-openshift]]
+= Install {product-title}
+:data-uri:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[install-overview]]
+== Overview
+
+This guide introduces you to the basic concepts of {product-title}, and helps
+you install a basic application. This guide is
+not suitable for deploying or installing a production environment of OpenShift.
+
+[[install-prerequisites]]
+=== Prerequisites
+
+To install {product-title}, you will need:
+
+* At least two physical or virtual RHEL 7+ machines, with fully qualified domain
+names (either real world or within a network) and password-less ssh access to
+each other. This guide, uses master.openshift.example.com and
+node.openshift.example.com. These machines must be able to ping each other
+using these domain names.
+* A valid Red Hat subscription.
+* Wildcard DNS resolution that resolves your domain to the IP of the node. So,
+an entry like the following in your DNS server:
+
+----
+master.openshift.example.com. 300 IN  A <IP of the master>
+node.openshift.example.com. 300 IN A <IP of the node>
+*.apps.openshift.example.com. 300 IN  A <IP of the node>
+----
+
+[NOTE]
+.Why the *apps* in your domain name for the wildcard entry?
+====
+When using {product-title} to deploy applications, an internal router needs to proxy
+incoming requests to the corresponding application pod. By using *apps* as part
+of the application domains, the application traffic is accurately marked to the
+right pod.
+
+You can use anything other than *apps*. For example,
+
+----
+*.cloudapps.openshift.example.com. 300 IN  A <IP of the node>
+----
+====
```

Comment:
Should the codeblock and "Why the apps..." section be tucked under the bullet above?